lib/bloc/reverse_swap_in/reverse_swaps_in_bloc.dart

```diff
@@ -0,0 +1,45 @@
+import 'dart:async';
+
+import 'package:breez_sdk/breez_bridge.dart';
+import 'package:breez_translations/breez_translations_locales.dart';
+import 'package:c_breez/bloc/reverse_swap_in/reverse_swaps_in_state.dart';
+import 'package:c_breez/utils/exceptions.dart';
+import 'package:fimber/fimber.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class ReverseSwapsInBloc extends Cubit<ReverseSwapsInState> {
```

Comment:
NIT: why `ReverseSwapInBloc` ? Maybe just `ReverseSwapBloc` ?

Reply:
I had the reverse swap related logic in`ReverseSwapBloc` before I merged them under `WithdrawFundsBloc` and the naming `SwapInBloc` was used for swaps in progress, that's why I had picked it. 

`ReverseSwapBloc` & `ReverseSwapsInProgressBloc`'s have to be separate if we need to periodically fetch in progress reverse swap list only when we're on `ReverseSwapPage`.